sys/src/9/port/sysproc.c

```diff
@@ -666,6 +685,8 @@ execac(Ar0* ar0, int flags, char *ufile, char **argv)
 		up->procctl = Proc_toac;
 		up->prepagemem = 1;
 	}
+	DBG("up %p: plan 9 proc\n", up);
```

Comment:
It's not always a plan9 proc at this point

Reply:
ah right I had condition DBG on plan 9. I think I can drop this print now.

Thanks!